projects/assets/src/translations/en/payment.ts

```diff
@@ -15,7 +15,7 @@ export const payment = {
     },
     cardNumber: 'Card Number',
     expirationDate: 'Expiration Date',
-    securityCode: 'Security code (CVV)',
+    securityCode: 'CVV/CVC',
```

Comment:
Changing or removing translation keys or values is breaking change
see https://sap.github.io/spartacus-docs/breaking-changes/#avoiding-breaking-changes